### PR TITLE
doc: Remove latin from sdk-nrfxlib docs

### PR DESCRIPTION
**Pull request by divipillai**
NCSDK-12222
Remove Latin abbreviations from the
sdk-nrfxlib documents.

Signed-off-by: divya pillai <divya.pillai@nordicsemi.no>

### CODE REVIEW

File: nrf_modem/doc/ug_nrf_modem_porting_os.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ See :ref:`trace_output_function` for more information.
 * Initialize timers/threads.
 * Configure low priority Modem library scheduling IRQ (SoftIRQ).
 * Configure low priority trace scheduling IRQ (SoftIRQ).
-* Configure medium for trace (UART/SPI etc.).
+* Configure medium for trace (UART/SPI).
```
Comment (peknis) [inline nrf_modem/doc/ug_nrf_modem_porting_os.rst:48]:
```suggestion
* Configure medium for trace (for example, UART or SPI).
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by czeslawmakarski — APPROVED** [review]

File: nrf_modem/doc/CHANGELOG.rst [hunk 2]
```diff
@@ -234,7 +234,7 @@ bsdlib 0.5.0
 ************
 
 * bsd_irrecoverable_handler() has been removed.
-  The application no longer needs to implement it to receive errors during initialization, which are instead reported via bsd_init().
+  The application no longer needs to implement it to receive errors during initialization, which are instead reported through bsd_init().
```
[the remark this answers is not in the document]
Comment (melwee01) [reply nrf_modem/doc/CHANGELOG.rst:237]: Do those render differently in the live doc?

File: nrf_rpc/doc/architecture.rst [hunk 3]
```diff
@@ -67,7 +67,7 @@ The following image presents sample command and response flows.
 
    nRF RPC simple command flow
 
-A thread waiting for a response can be reused to receive a new incoming command from the remote thread that the local thread is waiting for, e.g. when a callback is called synchronously.
+A local thread waiting for a response can be reused to receive a new incoming command from the remote thread, for example, when a callback is called synchronously.
```
Comment (doki-nordic) [inline nrf_rpc/doc/architecture.rst:70]: Removed part of the sentence specifies which remote thread can reuse the local thread. In modified version it is not clear which remote thread is allowed to reuse local, so the someone, who reads it, may think that any remote thread can reuse the local thread.

**Review by doki-nordic — CHANGES_REQUESTED** [review]

**Review by KAGA164 — APPROVED** [review]

File: nrf_modem/doc/CHANGELOG.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (divipillai) [reply nrf_modem/doc/CHANGELOG.rst:237]: After the `nrf_modem 0.8.99' release, bsdlib was renamed to Modem library (nrf_modem) and all the functions were also renamed. Hence, old functions are not linkable and render bit differently in the live docs.

_nrf_modem 0.8.99
Renamed from bsdlib to Modem library (nrf_modem)._

File: nrf_rpc/doc/architecture.rst (hunk 3 above)
Comment (divipillai) [reply nrf_rpc/doc/architecture.rst]: Rephrased as - A thread waiting for a response can be reused to receive a new incoming command from the remote thread for which the local thread is waiting, for example, when a callback is called synchronously.

**Review by peknis — APPROVED** [review]

File: nrf_rpc/doc/architecture.rst (hunk 3 above)
Comment (doki-nordic) [reply nrf_rpc/doc/architecture.rst]: That looks good.

**Review by doki-nordic — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]

**Review by frkv — APPROVED** [review]
LGTM

**Pull request merged**